lst_scripts/onsite_mc_r0_to_dl1.py

```diff
@@ -115,66 +162,123 @@ def main():
+    jobids_r0_dl1 = []
 
-    for l in 'training', 'testing':
-        if l == 'training':
+    for set_type in 'training', 'testing':
+        if set_type == 'training':
             list = training_list
         else:
             list = testing_list
-        dir_lists = os.path.join(RUNNING_DIR, 'file_lists_' + l)
+        dir_lists = os.path.join(RUNNING_DIR, 'file_lists_' + set_type)
         output_dir = os.path.join(RUNNING_DIR, 'DL1')
-        output_dir = os.path.join(output_dir, l)
-        check_and_make_dir(dir_lists)
-        check_and_make_dir(output_dir)
-        print("output dir: ", output_dir)
+        output_dir = os.path.join(output_dir, set_type)
+        if flag_full_workflow:
+            check_and_make_dir_without_verification(dir_lists)
+            check_and_make_dir_without_verification(output_dir)
+        else:
+            check_and_make_dir(dir_lists)
+            check_and_make_dir(output_dir)
+        print("\toutput dir: \t", output_dir)
 
         number_of_sublists = len(list) // NFILES_PER_DL1 + int(len(list) % NFILES_PER_DL1 > 0)
         for i in range(number_of_sublists):
-            output_file = os.path.join(dir_lists, '{}_{}.list'.format(l, i))
+            output_file = os.path.join(dir_lists, '{}_{}.list'.format(set_type, i))
             with open(output_file, 'w+') as out:
                 for line in list[i * NFILES_PER_DL1:NFILES_PER_DL1 * (i + 1)]:
                     out.write(line)
                     out.write('\n')
-        print('{} files generated for {} list'.format(number_of_sublists, l))
+        print('\t{} files generated for {} list'.format(number_of_sublists, set_type))
 
         ### LSTCHAIN ###
         counter = 0
 
         for file in os.listdir(dir_lists):
-            jobo = os.path.join(JOB_LOGS, "job{}.o".format(counter))
-            jobe = os.path.join(JOB_LOGS, "job{}.e".format(counter))
-            cc = ' -conf {}'.format(args.config_file) if args.config_file is not None else ' '
+            if set_type == 'training':
+                jobo = os.path.join(JOB_LOGS, "job{}_train.o".format(counter))
+                jobe = os.path.join(JOB_LOGS, "job{}_train.e".format(counter))
+            else:
+                jobo = os.path.join(JOB_LOGS, "job{}_test.o".format(counter))
+                jobe = os.path.join(JOB_LOGS, "job{}_test.e".format(counter))
+            cc = ' -conf {}'.format(config_file) if config_file is not None else ' '
             base_cmd = 'core_list.sh "lstchain_mc_r0_to_dl1 -o {} {}"'.format(output_dir, cc)
-            cmd = 'sbatch -e {} -o {} {} {}'.format(jobe, jobo, base_cmd, os.path.join(dir_lists, file))
 
-            # os.system(cmd)
-            print(cmd)
+            # recover or not the jobid depending of the workflow mode
+            if not flag_full_workflow:
+                cmd = 'sbatch -e {} -o {} {} {}'.format(jobe, jobo, base_cmd, os.path.join(dir_lists, file))
+
+                print(cmd)
+                # os.system(cmd)
```

Comment:
default ?

Reply:
yes, I think as it was originally. (maybe to avoid launching 200 jobs by error when testing the code ?)